Issue #25: System / Enhance Filesystem Utilization

NassimBtk opened the pull request:
- Enhanced the PromQL query to include the filesystem utilization percentage.
- Added table legends to the network, filesystem utilization and disk activity graphs and displayed the last* value for each legend.
- Left aligned filesystem paths in the Filesystem tables.
- Updated unit in the legend of the disk activity graph.
- Removed extra whitespace from the PromQL used to report the disk activity graph.

-------------------------------
### Testing
![image](https://github.com/user-attachments/assets/d68e4f1f-e4dd-482e-978f-348818530150)
![image](https://github.com/user-attachments/assets/3fd5d59c-508b-4ed8-8a7e-0aee6bccccff)
![image](https://github.com/user-attachments/assets/dbf86051-627b-4978-a8f7-05391083fd55)

NassimBtk committed:
Issue #25: System / Enhance Filesystem Utilization

- Enhanced the PromQL query to include the
  filesystem utilization percentage.
- Added table legends to the network, filesystem
  utilization and disk activity graphs
  and displayed the last* value for each legend.
- Left aligned filesystem paths in the Filesystem
  tables.
- Updated unit in the legend of the disk
  activity graph.
- Removed extra whitespace from the PromQL used
  to report the disk activity graph.

NassimBtk commented:
Thank you @MarcDouguedroit for your help.

bertysentry commented on src/dashboards/System/System (MetricsHub).json:
```diff
@@ -2839,7 +2877,7 @@
     }
   ],
-  "refresh": "1m",
+  "refresh": "5s",
```
This is probably way to short! 😅

NassimBtk commented on src/dashboards/System/System (MetricsHub).json:
```diff
@@ -2839,7 +2877,7 @@
     }
   ],
-  "refresh": "1m",
+  "refresh": "5s",
```
```suggestion
  "refresh": "30s",
 ```

NassimBtk replied:
We set the refresh to 30s to be consistent with 'MetricsHub Main' 😉

NassimBtk committed:
Issue #25: System / Enhance Filesystem Utilization #26

- Update src/dashboards/System/System (MetricsHub).json
- Set refresh to 30s to be consistent with 'MetricsHub Main'
